common/tablefunctions.lua

```diff
@@ -230,6 +230,23 @@ if not table.shuffle then
 	end
 end
 
+if not table.dump then
+	---Turns a table into a string
+	---@param o table
+	function table.dump(o)
```

Comment:
there's gotta be a `tostring` for tables implemented somewhere already. for example https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/gamedata/post_save_to_customparams.lua#L23

Reply:
https://github.com/beyond-all-reason/Beyond-All-Reason/blob/5e821f9124e54e9baf1078fd62ac194e4c099f6c/common/tablefunctions.lua#L69-L98

Reply:
well that's just embarrassing